TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bukkit/listeners/ServerRegister.java

```diff
@@ -0,0 +1,15 @@
+package cloud.timo.TimoCloud.bukkit.listeners;
+
+import cloud.timo.TimoCloud.api.events.EventHandler;
+import cloud.timo.TimoCloud.api.events.Listener;
+import cloud.timo.TimoCloud.api.events.server.ServerRegisterEvent;
+import cloud.timo.TimoCloud.bukkit.TimoCloudBukkit;
+
+public class ServerRegister implements Listener {
+
+    @EventHandler
+    public void onServerRegister(ServerRegisterEvent event) {
+        if (!event.getServer().getId().equals(TimoCloudBukkit.getInstance().getServerId())) return;
+        TimoCloudBukkit.getInstance().setServerRegistered(true);
```

Comment:
Why do we even need that? Can't we simply set the "registered" property to `true` as soon as the `registerAtCore()` method in `TimoCloudBukkit` has been called? And that's basically in the `onEnable` method...